### PR TITLE
Styling changes

### PR DESCRIPTION
**Pull request by MRFolio**

### CODE REVIEW

File: components/Misc/Alert.module.scss [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
+  font: {
+      size: 1.1rem;
+      weight: 700;
+  }
 }
 
 .alert-error {
-    background-color: rgba($color: red, $alpha: .3);
-    border: 1px solid red;
+  background-color: rgba($color: red, $alpha: 0.6);
```
Comment (niceFont) [inline components/Misc/Alert.module.scss]: Why did you change the opacity from 0.3 to 0.6?

File: styles/globals.scss [hunk 2]
```diff
@@ -26,63 +32,78 @@ $rounded-values: (
...
+    justify-content: center;
+    align-items: center;
+}
+
+@mixin flex-column {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
 }
```
Comment (niceFont) [inline styles/globals.scss]: Adding css to center it could be confusing since the mixin name just states that it's flex-column. Either renaming it to something like ```flex-col-center``` and ```flex-row-center``` could work or remove the centering part. Removing the centering part would make atleast one of the mixins pretty useless since it doesn't really save a lot of time/space though or what do you think?

**Review by niceFont — CHANGES_REQUESTED** [review]

File: components/Misc/Alert.module.scss (hunk 1 above)
Comment (MRFolio) [reply components/Misc/Alert.module.scss]: It makes the alert look more like an alert.

Comment (niceFont) [reply components/Misc/Alert.module.scss]: alrighty

**Review by niceFont — APPROVED** [review]
